src/lemonade/version.py

```diff
@@ -1 +1 @@
-__version__ = "9.0.6"
+__version__ = "10.0.0"
```

Comment:
```suggestion
__version__ = "9.1.0"
```

same version comment